dadapy/metric_comparisons.py

```diff
@@ -295,10 +303,16 @@ def greedy_feature_selection_full(self, n_coords, k=1, n_best=10, symm=True):
     def greedy_feature_selection_target(
         self, target_ranks, n_coords, k, n_best, symm=True
     ):
-        """Greedy selection of the set of coordinates which is most informative about a target distance.
+        """Greedy selection of the set of features which is most informative about a target distance.
+
+           Using the n-best single features describing the target_ranks, one more of all other features is added
+           combinatorically to make a candidate pool of duplets. Then, using the n-best duplets describing the
+           target_ranks, one more of all other features is added to make a candidate pool of triplets, etc.
+           This procedure is done until including the desired number of variables (n_coords) is reached.
 
         Args:
-            target_ranks (np.ndarray(int)): an array containing the ranks in the target space
+            target_ranks (np.ndarray(int)): an array containing the ranks in the target space, could be e.g.
+            the nearest neighbor ranks for a different set of variables on the same data points.
```

Comment:
same here, there should be enough space before "the nearest" in oder to let the automatic documentation builder understand that the line does not correspond to another parameter.

You should have

Args:
        target_ranks (np.ndarray(int)): description line 1
                description line 2

Reply:
Great, thanks. I relied on black formatting to do that correctly but no :/. Now it's aligned, I merge!

Reply:
Unfortunately black it's not that smart (yet :)